services/ext/api.go

```diff
@@ -814,6 +814,10 @@ func (api *PublicAPI) MarkAllRead(ctx context.Context, chatID string) error {
 	return api.service.messenger.MarkAllRead(ctx, chatID)
 }
 
+func (api *PublicAPI) MarkCommunityNotificationRead(ctx context.Context, communityID string) error {
```

Comment:
The name of this endpoint was confusing to me because we already have an endpoint `MarkActivityCenterNotificationsRead` which does essentially one thing, set `read = 1` in the notification. But `MarkCommunityNotificationRead` is doing more than just marking as read.

I suggest we use another name, for instance `DismissActivityCenterNotificationsByCommunity`, exactly the same name as in the `messenger`.